glider_kinematics_gazebo_plugins/kinematics_plugins/include/kinematics_plugins/KinematicsPlugin.hh

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2016 The UUV Simulator Authors.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// \file kinematics_plugins.hh
+/// \brief Plugin that for the vehicle kinematics
+
+#ifndef __KINEMATICS_PLUGIN_HH__
+#define __KINEMATICS_PLUGIN_HH__
+
+#include <map>
+#include <cmath>
+#include <string>
+
+#include <gazebo/gazebo.hh>
+#include <gazebo/transport/TransportTypes.hh>
+#include <kinematics_plugins/GaussMarkovProcess.hh>
+#include <sdf/sdf.hh>
+
+namespace gazebo
```

Comment:
Would help to include more documentation here.  See the API documentation in https://github.com/osrf/vrx/blob/master/usv_gazebo_plugins/include/usv_gazebo_plugins/acoustic_pinger_plugin.hh

Especially the documentation that explains the SDF parameters.